src/sharedHooks/useSuggestions/useShouldFetchOfflineSuggestionsUnconditionally.ts

```diff
@@ -0,0 +1,22 @@
+import { useMemo } from "react";
+
+const simultaneousOnlineOfflineSuggestionsEnabled = false;
+const simultaneousOnlineOfflinePercentageChance = 1;
+
+const useShouldFetchOfflineSuggestionsUnconditionally = ( ) => {
+  const fetchOfflineSuggestionsUnconditionally = useMemo(
+    () => {
+      if ( !simultaneousOnlineOfflineSuggestionsEnabled ) {
+        return false;
+      }
+      return Math.random() <= simultaneousOnlineOfflinePercentageChance;
```

Comment:
`Math.random() * 100` ? Although I am also fine with using 0.01 for `simultaneousOnlineOfflinePercentageChance` even though it is labelled as percentage.

Reply:
Yes, you're correct, thank you! I renamed / changed the "concept" of the constant (on the 1 vs 100 scale) after testing and just before push ing and I wasn't careful enough with it.